src/components/Media/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Config from 'config';
+
+import Navbar from 'components/Navbar';
+import Button from 'components/Button';
+import Footer from 'components/Footer';
+import Banner from 'components/Banner';
+
+export default class Media extends React.Component {
+	render() {
+		return (
+			<div className="media-page">
+			<Navbar />
+			<Banner decorative />
+			<div className="content-section center">
+				<h2> Media </h2>
```

Comment:
Remove spaces around media. Generally, html is structured like this `<h2>Text</h2>` without spaces. We don't want the spaces to render on the page, so remove them. This is also the convention that pretty much all web developers use.